source/less/blocks/control.less

```
@@ -1,6 +1,97 @@
 .control {
+  position: absolute;
+  width: 87.5%;
```

такой вариант не подходит потому что при чуть большей ширине экрана > 320px, будут разные отступы слева и справа
 ```suggestion
  width: calc(100% - 40px);
```